packages/adaptor-docs/src/DocsPanel.tsx

```diff
@@ -23,9 +23,9 @@ const DocsPanel = ({ specifier, onInsert }: DocsPanelProps) => {
   
   const { name, version, functions } = pkg as PackageDescription;
   return (
-    <div className="block m-2">
-      <h1 className="h1 text-lg font-bold text-secondary-700 mb-2">{name} v{version}</h1>
-      <div className="text-sm mb-4">Operations available for use by this adaptor are listed below.</div>
+    <div className="ad--block ad--m-2">
```

Comment:
This is also part of the actual fix - repeated across both components in the app.